core/src/main/java/tc/oc/pgm/projectile/ProjectileMatchModule.java

```diff
@@ -105,6 +106,9 @@ && isValidProjectileAction(event.getAction(), projectileDefinition.clickAction))
               player.getWorld().spawn(player.getEyeLocation(), projectileDefinition.projectile);
           projectile.setVelocity(velocity);
         }
+        if (projectileDefinition.yield != null && projectile instanceof Explosive) {
+          ((Explosive) projectile).setYield(projectileDefinition.yield);
```

Comment:
Indention off?

Reply:
This was run through the formatter so the indentation is correct.